setup.md

```diff
@@ -1,7 +1,19 @@
 ---
 title: Setup
 ---
-FIXME
+# Python
+For this lesson we will be using Python 3, the lessons are created assuming at least version 3.7.
```

Comment:
We should probably include the default carpentries instructions for installing python with jupyter lab.